src/mrinufft/operators/interfaces/base.py

```diff
@@ -103,10 +110,21 @@ def adj_op(self, coeffs):
         """
         pass
 
+    def with_off_resonnance_correction(self, B, C, indices):
```

Comment:
It can be both. But I like the natural formalism of getting the augmented operator from the vanilla one. Anyway off-resonnance is half supported for now, so thats a debate that can be post-poned ;)

Reply:
Haha cool, resolving